sympy/polys/domains/expressiondomain.py

```diff
@@ -9,6 +9,9 @@
 from sympy.polys.polyutils import PicklableWithSlots
 from sympy.utilities import public
 
+eflags = dict(mul=True, power_exp=False, power_base=False,
```

Comment:
The first attribute could be `deep=False` to save time. It is not necessary to inspect anything but the topmost node as this `simplify` method will be called repeatedly on each step.